packages/esm-service-queues-app/src/patient-search/visit-form-queue-fields/visit-form-queue-fields.component.tsx

```diff
@@ -105,59 +102,40 @@ const StartVisitQueueFields: React.FC = () => {
         )}
       </section>
 
```

Comment:
Is this among the dead code you are referring to?  I would have thought that we'd want to allow the user to choose the status, while providing a default, but I don't see this happening on dev3

Reply:
Yeah, that `styles.sectionHidden` makes it not show up. And it's not toggled anywhere. Not sure why someone would have decided to make it not appear on the screen, rather than deleting it. Worse than commented-out code.

Reply:
Oops, the status part of the form was being used, passed through the form submit event, which made it not subject to type-checking. I have fixed this in a follow-up PR: https://github.com/openmrs/openmrs-esm-patient-management/pull/1171